analytics/views.py

```diff
@@ -1125,57 +1127,72 @@ def support(request: HttpRequest) -> HttpResponse:
             current_plan_type = realm.plan_type
             do_change_plan_type(realm, new_plan_type)
             msg = f"Plan type of {realm.string_id} changed from {get_plan_name(current_plan_type)} to {get_plan_name(new_plan_type)} "
-            context["message"] = msg
+            context["success_message"] = msg
         elif request.POST.get("discount", None) is not None:
             new_discount = Decimal(request.POST.get("discount"))
             current_discount = get_discount_for_realm(realm)
             attach_discount_to_realm(realm, new_discount)
             msg = f"Discount of {realm.string_id} changed to {new_discount} from {current_discount} "
-            context["message"] = msg
+            context["success_message"] = msg
+        elif request.POST.get("new_subdomain", None) is not None:
+            new_subdomain = request.POST.get("new_subdomain")
+            old_subdomain = realm.string_id
+            try:
+                get_realm(new_subdomain)
+            except Realm.DoesNotExist:
+                do_change_realm_subdomain(realm, new_subdomain)
+                context["success_message"] = f"Subdomain changed from {old_subdomain} to {new_subdomain}"
+                context["redirect_to"] = "/activity/support?" + urlencode({"q": new_subdomain})
+            else:
+                if new_subdomain == old_subdomain:
+                    error_message = "The new subdomain should not be same as the old subdomain."
```

Comment:
"The new subdomain should not be same as the old subdomain." -> "The new subdomain should not be **the** same as the **current** subdomain."